src/pages/ProtectedRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate } from 'react-router-dom'
```

Comment:
Create top lvl folder `routes` where you will have `routes/routes.tsx` and `routes/ProtectedRoute.tsx`